2-dom-manipulation/form.js

```diff
@@ -0,0 +1,65 @@
+let form = document.querySelector('form');
+
+function structureIf(nameCondition){
+    if (nameCondition.value == '') {
+        console.log(`${nameCondition.id}: It's need value`);
+        nameCondition.classList.add('required');
```

Comment:
You can save the class name in a const.
`const REQUIRED_CLASS = 'required';

Reply:
I would name it `const ERROR_CLASS = 'error';`.
Because I can enter a value, but with the wrong format. So it would be a format problem and not a required problem.